openpype/hosts/max/plugins/publish/collect_review.py

```diff
@@ -0,0 +1,93 @@
+# dont forget getting the focal length for burnin
+"""Collect Review"""
+import pyblish.api
+
+from pymxs import runtime as rt
+from openpype.lib import BoolDef
+from openpype.pipeline.publish import OpenPypePyblishPluginMixin
+
+
+class CollectReview(pyblish.api.InstancePlugin,
+                    OpenPypePyblishPluginMixin):
+    """Collect Review Data for Preview Animation"""
+
+    order = pyblish.api.CollectorOrder + 0.02
+    label = "Collect Review Data"
+    hosts = ['max']
+    families = ["review"]
+
+    def process(self, instance):
+        nodes = instance.data["members"]
+        focal_length = None
+        camera = None
+        for node in nodes:
+            if rt.classOf(node) in rt.Camera.classes:
+                rt.viewport.setCamera(node)
```

Comment:
Isn't it weird to set the viewport camera in the collector - what's the reason we need to?